articles/multifactor-authentication/guardian/configuration.md

```diff
@@ -0,0 +1,63 @@
+# Setting Up Guardian
+
+Guardian is Auth0's multifactor authentication (MFA) application that provides a simple, safe way for you to implement MFA. The Guardian app is currently available for mobile devices running iOS or Android.
+
+For applications where Guardian MFA is enabled, the user will be required to sign in **and** confirm the logon with a verified mobile device. You can find additional information on the user log in process [here](#).
+
+## Implementing Multifactor Authentication
+Within Auth0, you may implement MFA via the **Multifactor Auth** page of the Management Dashboard.
+
+![](/media/articles/mfa/guardian-dashboard.png)
+
+> Auth0 provides [built-in support](https://auth0.com/docs/multifactor-authentication#using-auth0-s-built-in-support) for MFA using Google Authenticator or Duo. You may choose to use either of these providers, in lieu of Guardian, on the **Multifactor Auth** page of the Management Dashboard.
+
+### Configuring Guardian in the Management Dashboard
+
+The first thing you will do when setting up Guardian is to decide whether you would like MFA to occur via push notifications, SMS, or both.
+
+* **Push Notifications**: the user receives, via the Guardian app, a push notification that requires their input prior to gaining access to the app;
+* **SMS**: the user receives, via SMS, a code that they are required to enter prior to gaining access to the app.
+
+To enable either Push Notifications or SMS verification, move the appropriate slider to the right.
+
+![](/media/articles/mfa/guardian-both.png)
+
+Once you have enabled either option, you will be presented with the **Customize MFA** code snippet you will need to edit to ensure that MFA is applied to the appropriate Clinets. You will need the appropriate `clientID` values, and the code is executed as part of a [Rule](/rule) whenever a user logs in.
```

Comment:
a minor typo: Clinets
Also defaults enables you to use Guardian, so it is not a "must" it is an option, we are "flexible", but we also provide a frictionless aproach